tsconfig.json

```diff
@@ -2,6 +2,7 @@
   "extends": "./tsconfig.build.json",
   "compilerOptions": {
     "baseUrl": "./packages",
+    "types": ["cypress"],
```

Comment:
Putting this here makes WebStorm go a bit crazy. When I open one of the files in the `cypress` folder it complains that its not under any tsconfig, and then outputs hundreds of errors not finding built-in things of the DOM API like `Node` or `Array`.
I think it would be better to add a tsconfig file to this folder too and inherit from this one.

Reply:
And to import types from Cypress explicitly instead of adding them globally

Reply:
I don't even think that part is needed anymore. I was trying to solve some issue but that wasn't the right solution and I just left it there accidentally. Can you please check if it works for you now?